src/algorithm/verifySolidAccessToken.ts

```diff
@@ -30,10 +30,12 @@ export async function verifySolidAccessToken(
 ): Promise<SolidAccessTokenPayload> {
   // Extract access token value from authorization header
   const accessTokenValue = authorization.header.replace(/^(DPoP|Bearer) /, "");
+  const accessTokenParts =
+    /(?<=^\w+\.)(\w+).(\w+)/.exec(accessTokenValue) ?? [];
```

Comment:
I need a bit of time to ponder this regex, but I just realised that there is an actual error in the previous one while checking the ABNF there: https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-04#section-7.1

We need to match one or more spaces on the header after the token type string (not one). I'll create an issue because we need to add tests for that.

Reply:
OK, so interestingly, Panva which is the creator of jose and all the node oidc libraries recommends using a simple split for the access token segments: https://github.com/panva/jose/discussions/106#discussioncomment-210262

I think I would feel more comfortable sticking to that unless there is a specific reason to use a more complex approach.

Reply:
Agreed.